inlang/source-code/manage/tsconfig.json

```diff
@@ -11,5 +11,9 @@
 		"outDir": "./dist",
 		"rootDir": "./src",
 		"resolveJsonModule": true
+		// Don't check imported libraries for type errors.
+		// The imported libraries might have different settings/whatever.
+		// "skipLibCheck": false,
```

Comment:
remove obesolete comments

Reply:
I should have marked this pr back to draft - commited a WIP state to check ci - thanks for raising